projects/freerdp/project.yaml

```diff
@@ -0,0 +1,16 @@
+homepage: "https://www.freerdp.com/"
+language: c
+primary_contact: ""
+builds_per_day: 4
+auto_ccs:
+  - "estetus@gmail.com"
+fuzzing_engines:
+  - afl
+  - honggfuzzer
+  - libfuzzer
+sanitizers:
+  - address
+  - memory
```

Comment:
"memory" sanitizer is generally hard to get working, unlike "undefined" behavior sanitizer, is there a reason why you don't want to try getting "undefined" working?

Reply:
replaced `memory` with `undefined`